backend/main.py

```
@@ -0,0 +1,62 @@
+    allow_origins=settings.cors_origins,
+    allow_credentials=settings.cors_allow_credentials,
+    allow_methods=settings.cors_allow_methods,
+    allow_headers=settings.cors_allow_headers,
+)
+
+# Include API routes
+app.include_router(api_router_v1, prefix="/api/v1")
+
+
+@app.get("/")
+async def root():
```

Ruff (0.12.2)

Missing return type annotation for public function `root`

(ANN201)